output.tf

```diff
@@ -2,3 +2,8 @@ output record_name {
   value       = aws_route53_record.record.fqdn
   description = "The FQDN of the resulting DNS record."
 }
+
+output bucket_arn {
+  value       = aws_s3_bucket.bucket_index[0].arn
```

Comment:
@mgroeningEC ich glaub du bekommst hier ein Problem für den Fall das kein Bucket `aws_s3_bucket.bucket_index[0]` existiert auf Grund `count = 0`, dann müsste ja das andere Bucket greifen?

Du kannst dir das als konkretes Beispiel beide Fälle probieren in einem neuen dummy Projekt und dort einfach ein `terraform plan` ausführen, also für beide Beispiele. Ich gehe davon aus, dass du bereits hier dann beim Plan ein Error bekommst. Für das lokal ausprobieren, kannst du auch einfach sowas machen:

```hcl
module "edithcare_maintenance_api" {
  source = "/Users/david/Development/synlay/customers/edith.care/ops/terraform-aws-maintenance-mode-api"
```

Reply:
danke für den Hinweis! probier ich sofort mal aus